# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/tests/test_bug1314790.html
# Commit: 366245552cae
# Full Hash: 366245552cae536ef883be806322395dc47b759d
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2025-06-17 09:24:05
# Regressor Bug: 1951038
# File Overlap Count: 1
# Description:
#   Bug 1951038 - Get rid of the legacy white-space normalizer of `HTMLEditor` and the pref r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D253589
# ==============================================================================

diff -r b13b0a530402 -r 366245552cae editor/libeditor/tests/test_bug1314790.html
--- a/editor/libeditor/tests/test_bug1314790.html	Tue Jun 17 00:10:27 2025 +0000
+++ b/editor/libeditor/tests/test_bug1314790.html	Tue Jun 17 00:37:22 2025 +0000
@@ -22,9 +22,6 @@
 <script class="testbody" type="application/javascript">
 SimpleTest.waitForExplicitFinish();
 SimpleTest.waitForFocus(function() {
-  const kNewWhiteSpaceNormalizerEnabled =
-    SpecialPowers.getBoolPref("editor.white_space_normalization.blink_compatible");
-
   let elm = document.getElementById("contenteditable1");
   elm.focus();
   window.getSelection().collapse(elm.childNodes[0], 0);
@@ -52,7 +49,7 @@
   SpecialPowers.doCommand(window, "cmd_deleteWordBackward");
   is(
     elm.childNodes[0].textContent,
-    kNewWhiteSpaceNormalizerEnabled ? "pen\u00A0" : "pen ",
+    "pen\u00A0",
     "'pen ' is first node"
   );
 